scss/components/hero-banner/hero-banner.scss

```diff
@@ -36,7 +36,11 @@ $hero-max-width-lg: 1500px;
   background-color: color(charcoal);
   background-size: cover;
   position: relative;
-  margin-top: em(-65px);
+  margin-top: -121px;
```

Comment:
Did you intend to include hero-banner scss changes in the logo-lockup? If not please revert.